src/pages/MainPage/HeroSection.jsx

```diff
@@ -1,18 +1,22 @@
 import SearchPannel from '../../components/common/SearchPannel/SearchPannel';
+import BackgroundHero from '../../assets/img/mainPage/hero.png';
 import './heroSection.scss';
 
 const HeroSection = () => {
   return (
-    <section className="hero">
-      <div className="hero-wrapper">
-        <div className="hero-info">
-          <h1 className="hero-info-title">
+    <section
+      className="... min-h-[40.75rem] bg-cover bg-center bg-no-repeat"
+      style={{ backgroundImage: `url(${BackgroundHero})` }}
+    >
+      <div className="container mx-auto my-0 flex justify-end pt-24">
+        <div className="relative flex max-w-[40rem] flex-col">
+          <h1 className="font-third mb-40 self-end text-right text-5xl font-normal not-italic text-light-color">
             Discover and share travel guides from around the world
           </h1>
-          <div className="hero-info-search">
+          <div className="absolute right-14 top-2/3 translate-y-[-50%]">
```

Comment:
Why do you need absolute positioning here?

Reply:
![image](https://github.com/voyageglobal/voyage-frontend/assets/170460349/bb47e3af-40ce-4e7f-9789-dfc717d4bb2c)
![image](https://github.com/voyageglobal/voyage-frontend/assets/170460349/66a114be-9e79-4404-b5a7-5d6958d0ff38)

It's not a grid-element, so I can't control it's positioning otherwise. If I use flex, then this is neither the beginning nor the end of the container, and even nor the center.